content/en/docs/installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-do.md

```diff
@@ -25,29 +26,27 @@ You need to select:
 
 ![config-cluster-do](/images/docs/do/config-cluster-do.png)
 
-{{< notice note >}} 
+{{< notice note >}}
 
 - Supported Kubernetes versions for KubeSphere 3.0.0: 1.15.x, 1.16.x, 1.17.x, 1.18.x.
 - 2 nodes are included in this example. You can add more nodes based on your own needs especially in a production environment.
-- The machine type Standard / 4 GB / 2 vCPUs is for minimal installation. If you plan to enable several pluggable components or use the cluster for production, you can upgrade your nodes to a more powerfull type (such as CPU-Optimized / 8 GB / 4 vCPUs). It seems that DigitalOcean provisions the master nodes based on the type of the worker nodes, and for Standard ones the API server can become unresponsive quite fast.
+- The machine type Standard / 4 GB / 2 vCPUs is for minimal installation. If you plan to enable several pluggable components or use the cluster for production, you can upgrade your nodes to a more powerfull type (such as CPU-Optimized / 8 GB / 4 vCPUs). It seems that DigitalOcean provisions the master nodes based on the type of the worker nodes, and for Standard ones the API server can become unresponsive quite soon.
 
-{{</ notice >}} 
+{{</ notice >}}
 
 When the cluster is ready, you can download the config file for kubectl.
 
 ![download-config-file](/images/docs/do/download-config-file.png)
 
 ## Install KubeSphere on DOKS
 
-Now that the cluster is ready, you can install KubeSphere following the steps below:
+Now the cluster is ready, you can install KubeSphere following the steps below:
```

Comment:
"now that" is a set phrase, equivalent to the word "since". If you delete it, the sentence has a [comma splice](https://en.wikipedia.org/wiki/Comma_splice) issue. If you think the sentence is not clear enough, change it to:

As the cluster is ready now, you can install KubeSphere following the steps below:

Reply:
OK, I see. I am not sure if it is correct or not to use now that or since here in terms of context. I will revert it.